src/components/Input/Input.tsx

```diff
@@ -0,0 +1,36 @@
+import { dropdownData } from "../../core/config/constants";
+import Dropdown from "../Dropdown/Dropdown";
+
+const InputField = ({
+  description,
+  isMandatory,
+  inputType,
+}: {
+  description: string;
+  isMandatory: boolean;
+  inputType: string;
+}) => {
+  return (
+    <label className="flex flex-column">
+      <div>
+        {isMandatory && <span className="asterisk">*</span>}
+        <span> {description}:</span>
+      </div>
+      {inputType?.split(":")[0] == "custom" ? (
+        <Dropdown
+          inputField={inputType?.split(":")[1]}
+          options={
+            dropdownData[
```

Comment:
Instead of this add a new prop for dropdown data

Reply:
Correction done in Phase-2 branch.